### PR TITLE
Pathology page

### PR DESCRIPTION
**Pull request by sharlagelfand**
First pass at the pathology page of the app (in a very basic Shiny app), with a plot to explore the different phenotypes.

Includes the ability to filter by a single phenotype and tissue, and select multiple mouse lines (just one available for now, but I have some [test cases](https://github.com/Sage-Bionetworks/mouse-agora/blob/phenotypes/tests/testthat/test-plot_phenotypes.R#L43) ([rendered](https://github.com/Sage-Bionetworks/mouse-agora/blob/phenotypes/tests/figs/plot-phenotypes/two-row-facets.svg)) of what it'll look like when multiple are selected). When there's no data for either the experiment/control line, there's a message on the facet indicating so. There aren't any cases of that with the current data, but you can also see what it looks like in a [test](https://github.com/Sage-Bionetworks/mouse-agora/blob/phenotypes/tests/testthat/test-plot_phenotypes.R#L51) ([rendered](https://github.com/Sage-Bionetworks/mouse-agora/blob/phenotypes/tests/figs/plot-phenotypes/no-data-control.svg))

I didn't commit the raw data file (or the corresponding processed `.rda`) file, so you'll need to put the xlsx file in `data-raw/` and run [`phenotypes.R`](https://github.com/Sage-Bionetworks/mouse-agora/blob/phenotypes/data-raw/phenotypes.R) before you can run the app locally. If these can be put on GitHub let me know, but I figured not for now.

Depending on how prescriptive some of the plotting elements outlined are (e.g. labelling `Age` on the x-axis on each plot, legend inside the plot rather than beside, etc), I think patchworked plots could work better than the current faceted approach, but that could add complications with other things (e.g. consistent limits, boxplot widths, etc) so I'd be curious to know what you think.

I haven't yet handled the case where there's no data for both the control and experiment facets for all lines so e.g. Phenotype = "Plasma AB 42" will error for now.

Also, happy to have any language (in the code or the app!) corrected - the domain is new to me.

Thanks!

### CODE REVIEW

File: R/plot_phenotypes.R [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#' @param data Input data (the result of \code{\link{filter_pathology}} and \code{\link{expand_mouse_line_factor}} with \code{\link{phenotypes}} as input).
+#'
+#' @return A ggplot2 object.
+#' @export
+plot_phenotypes <- function(data) {
+
+  # Generate annotation for mouse lines (facets) that won't have any data
+  measured_annotation <- data %>%
+    dplyr::count(.data$mouse_line_full, .drop = FALSE) %>%
+    dplyr::filter(n == 0) %>%
```
[the remark this answers is not in the document]
Comment (sharlagelfand) [reply R/plot_phenotypes.R:12]: Good catch, thanks!

File: DESCRIPTION [hunk 2]
```diff
@@ -0,0 +1,35 @@
+Package: mouseagora
```
[the remark this answers is not in the document]
Comment (sharlagelfand) [reply DESCRIPTION:1]: Sure, let's do `magora`! I'll change the package name

File: R/mod_pathology.R [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+          ns("phenotype"),
+          "Select a phenotype",
+          choices = unique(phenotypes[["phenotype"]])
+        ),
+        shinyWidgets::pickerInput(
+          ns("mouse_line"),
+          "Select mouse lines",
+          choices = unique(phenotypes[["mouse_line"]]),
+          multiple = TRUE,
+          selected = "BL6"
```
[the remark this answers is not in the document]
Comment (sharlagelfand) [reply R/mod_pathology.R:25]: Thanks, I'll update it! So users would be able to select only one of them, and e.g. if BL5 also exists, they could also select "BL6", ""5XfAD;BL6", "BL5" but not "5XfAD;BL6"? Trying to map out how these edge cases would look

**Comment by sharlagelfand** [comment]
Thanks @karawoo! I checked in the missing `inst/app/www` folder and removed variables that don't have data.

I don't mind the faceting but just wanted to check if some things in the visualization mockup are flexible - e.g. not just having consistent `Age` values on the x-axis, but also labelling each plot's x-axis with the "Age" title or labelling each y-axis with the different phenotype values (as opposed to it just appearing on the left plot). Otherwise I think faceting is a fine approach (and probably faster than generating and combining multiple plots?).

**Comment by sharlagelfand** [comment]
Now you can see my force push shame 🙈

File: DESCRIPTION (hunk 2 above)
[the remark this answers is not in the document]
Comment (karawoo) [reply DESCRIPTION]: 👍  I changed the repo name

File: R/mod_pathology.R (hunk 3 above)
[the remark this answers is not in the document]
Comment (karawoo) [reply R/mod_pathology.R:25]: Now I'm a little confused about the multiple selection. Let me double check with Anna about what she's envisioning here. We don't need to hold up this PR for it

**Comment by karawoo** [comment]
No shame!!

I think the mock-ups are quite flexible. Once we have this app deployed in a place where we can show it to Anna and our other collaborators, they may have some opinions about various aspects of the plot appearance, but I wouldn't worry too much about the small details of axis labeling etc. until then.

File: DESCRIPTION [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    sagethemes,
+    magrittr,
+    rlang
+Encoding: UTF-8
+LazyData: true
+RoxygenNote: 7.1.0
+Depends: 
+    R (>= 2.10)
+URL: https://github.com/Sage-Bionetworks/mouse-agora
+BugReports: https://github.com/Sage-Bionetworks/mouse-agora/issues
```
Comment (karawoo) [inline DESCRIPTION:30]:
```suggestion
URL: https://github.com/Sage-Bionetworks/magora
BugReports: https://github.com/Sage-Bionetworks/magora/issues
```

Comment (sharlagelfand) [reply DESCRIPTION]: Thanks!

**Comment by sharlagelfand** [comment]
Okay yes, good reminder not to sweat the small stuff too early on, thanks! Would you like to be the keeper of the merge or should I go for it?

**Comment by karawoo** [comment]
go for it!

**Pull request merged**